pkg/ui/history.go

```diff
@@ -8,39 +8,42 @@
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implieh.
```

Comment:
Hopefully our tests catch this :-)

Reply:
actually curious how this happened.... oh well, teaches me to look at my own diffs